modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+status: "True"
+type: InstanceExists
+
+lastTransitionTime: "2022-08-03T08:36:37Z"
+status: "True"
+type: Terminable
+lastUpdated: "2022-08-03T08:40:36Z"
+nodeRef:
+kind: Node
+name: openshift-control-plane-2
+uid: 788df282-6507-4ea2-9a43-24f237ccbc3c
+phase: Running
+----
+
+. Ensure that the Bare Metal Operator is available:
++
+[source,terminal]
+----
+$ oc get clusteroperator baremetal
+----
++
+.Example output
+[source,terminal]
+----
+NAME        VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+baremetal   4.11.3    True        False         False      3d15h
+----
+
+. Delete the machine of the unhealthy member:
++
+[source,terminal]
+----
+$ oc delete machine -n openshift-machine-api examplecluster-control-plane-2 <1>
+----
++
+If deletion of the machine is delayed for any reason or the command is obstructed and delayed, you can force deletion by removing the machine object finalizer field. 
++
+[IMPORTANT]
+====
+Do not interrupt machine deletion with `control-c`. You must allow the command to proceed to completion. Open a new terminal window to edit and delete the finalizer fields.
+====
++
+[source,terminal]
+----
+$ oc edit machine -n openshift-machine-api examplecluster-control-plane-2
+----
++
+.. Find and delete the fields:
++
+[source,terminal]
+----
+finalizers:
+- machine.machine.openshift.io
+----
++
+and save the change. 
++
+[source,terminal]
+----
+machine.machine.openshift.io/examplecluster-control-plane-2 edited
+----
++
+.. Verify the machine was deleted:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                              PHASE     TYPE   REGION   ZONE   AGE     NODE                                 PROVIDERID                                                                                       STATE
+examplecluster-control-plane-0    Running                          3h11m   openshift-control-plane-0   baremetalhost:///openshift-machine-api/openshift-control-plane-0/da1ebe11-3ff2-41c5-b099-0aa41222964e   externally provisioned
+examplecluster-control-plane-1    Running                          3h11m   openshift-control-plane-1   baremetalhost:///openshift-machine-api/openshift-control-plane-1/d9f9acbc-329c-475e-8d81-03b20280a3e1   externally provisioned
+examplecluster-compute-0          Running                          165m    openshift-compute-0         baremetalhost:///openshift-machine-api/openshift-compute-0/3d685b81-7410-4bb3-80ec-13a31858241f         provisioned
+examplecluster-compute-1          Running                          165m    openshift-compute-1         baremetalhost:///openshift-machine-api/openshift-compute-1/0fdae6eb-2066-4241-91dc-e7ea72ab13b9         provisioned
+----
+
+. Remove the old `BareMetalHost` object:
++
+[source,terminal]
+----
+$ oc delete bmh openshift-control-plane-2 -n openshift-machine-api
+
+baremetalhost.metal3.io "openshift-control-plane-2" deleted
```

Comment:
I think this should be separated into an example output

Reply:
Agreed. Done!